lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -130,6 +131,56 @@ def add_pods_library
           end
         end
 
+        # Add a reference to pods' resources (like xcassets) to the user target
+        #
+        # @return [Boolean] true if user project has been modified
+        #
+        def add_pods_resources
+          dirty = false
+        
+          UI.puts "- User targets being integrated: #{native_targets.map(&:name).inspect}"
+          UI.puts "- Pod Targets = #{target.pod_targets.map(&:name)}"
+          target.pod_targets.each do |pod_target|
+            pod_name = pod_target.pod_name
+            resource_files = pod_target.file_accessors.flat_map(&:resources)
+            resource_files.each do |resource_path|
+              UI.puts "   - Adding #{resource_path} to user project"
+
+              relative_path = resource_path.relative_path_from(target.client_root).to_s
+              pods_group = user_project['Pods'] || ((dirty = true) && user_project.new_group('Pods'))
+              resources_group = pods_group['Resources'] || ((dirty = true) && pods_group.new_group('Resources'))
+              pod_subgroup = resources_group[pod_name] || ((dirty = true) && resources_group.new_group(pod_name))
+              file_ref = pod_subgroup.files.find { |f| f.path == relative_path }
+              file_ref ||= (dirty = true) && pod_subgroup.new_file(relative_path)
+
+              UI.puts "   - Adding #{resource_path.basename} to targets #{native_targets.map(&:name)}"
+              native_targets.each do |user_target|
+                unless user_target.resources_build_phase.include?(file_ref)
+                  user_target.add_resources([file_ref])
+                  dirty = true
+                end
+              end              
+            end
+          end
+          # TODO: Remove/Deintegrate pods resources no longer in a target (remove from target,
+          #         or even from projet if no more target linked to)
+          # TODO: Remove code in FileReferencesInstaller#add_resource that adds resources to the Pods.xcodeproj
+          # TODO: Remove Pods-resources.sh (completely or partially?)
```

Comment:
Should we remove other stuffs as well?

* Like `xcdatamodeld` files probably have the same issue than `xcassets` in the end and could probably take advantage of this fix too?
* But is it true for every kind of resources here (like `*.framework` which are also handled specially in `Pods-resources.sh`)?